### PR TITLE
Add new resource "Service LB Policy" to the provider

### PR DESCRIPTION
**Pull request by Samir-Cit**
This PR adds the new resource **Service LB Policies** `google_network_services_service_lb_policies` to the providers.

- Once the API reference page is provided by Google team this PR will be updated.

**Release Note Template for Downstream PRs (will be copied)**

```release-note:new-resource
`google_network_services_service_lb_policies`
```

### CODE REVIEW

**Comment by Samir-Cit** [comment]
@c2thorn were you able to check this PR?

**Comment by c2thorn** [comment]
> * Once the API reference page is provided by Google team this PR will be updated.

Apologies, getting to this now.

> Once the API reference page is provided by Google team this PR will be updated.

Is this new enough that it doesn't have a reference page? Is the feature publicly available?

**Comment by Samir-Cit** [comment]
> > Once the API reference page is provided by Google team this PR will be updated.
> 
> Is this new enough that it doesn't have a reference page? Is the feature publicly available?

The resource have some documentation pages but the `API Discovery Doc` is not available yet. Apparently this doc was sent to review today.
For the development of the `Terraform resource` I used this doc regarding [set up advanced load balancing](https://cloud.google.com/service-mesh/docs/service-routing/set-up-advanced-load-balancing)

**Comment by Samir-Cit** [comment]
Hello @c2thorn 
I updated the PR, can you do a round of review?

The updates were:
- Merge main
- Test fix
- New example/test with backend_service

**Review by c2thorn — CHANGES_REQUESTED** [review]

File: mmv1/third_party/terraform/services/networkservices/resource_network_services_service_lb_policies_test.go [hunk 1]
```diff
@@ -0,0 +1,75 @@
+package networkservices_test
```
[the remark this answers is not in the document]
Comment (Samir-Cit) [reply mmv1/third_party/terraform/services/networkservices/resource_network_services_service_lb_policies_test.go:1]: Sure thing.
I updated the test following the example you sent and changed to `.go.erb`.
I run locally and it's working fine.

File: mmv1/products/networkservices/ServiceLBPolicies.yaml [hunk 2]
```diff
@@ -0,0 +1,115 @@
...
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an 'AS IS' BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+--- !ruby/object:Api::Resource
+name: 'ServiceLbPolicies'
+base_url: 'projects/{{project}}/locations/{{location}}/serviceLbPolicies'
```
[the remark this answers is not in the document]
Comment (Samir-Cit) [reply mmv1/products/networkservices/ServiceLBPolicies.yaml:16]: That totally makes sense.
I just added this `min_version: beta` on the provider and as soon as the official doc is released we can open other PR with the changes.

**Review by c2thorn — APPROVED** [review]

**Pull request merged**